src/navalmartin_mir_db_utils/examples/simple_queries.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+import bson
+from navalmartin_mir_db_utils.dbs.mongodb_session import MongoDBSession
+from navalmartin_mir_db_utils.crud.mongodb_crud_utils import ReadEntityBaseCRUDAPI
+
+
+IMAGES_COLLECTION_TO_READ = 'YOUR_COLLECTION_NAME'
+
+
+async def query_db(mongodb_session: MongoDBSession, criteria: dict,
+                   projection: dict,
+                   collection_name: str):
+    query_result = ReadEntityBaseCRUDAPI.do_find(criteria=criteria, projection=projection,
+                                                 db_session=mongodb_session,
+                                                 collection_name=collection_name)
+    images = [img async for img in query_result]
+    return images
+
+
+def main():
+    MONGODB_URL = "YOUR_MONGODB_URL"
```

Comment:
are these environmental variables or is it ok to hardcode some strings here.

Reply:
These are environment variables